Library/MapleBacon/MapleBacon/MapleBaconConfig.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  MapleBaconConfig.swift
+//  MapleBacon
+//
+//  Created by Danilo Topalovic on 10.03.16.
+//  Copyright © 2016 Zalando SE. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/**
+ * MapleBacon Config
+ */
+public class MapleBaconConfig {
+    
+    /// singleton param
```

Comment:
Same for this comment, it's pretty standard practice to name singletons `sharedSomething` in the iOS world. Also maybe this should be called `defaultConfiguration`, similar to what `URLSessionConfiguration` does?